scripts/Muon/GUI/FrequencyDomainAnalysis/plot_widget/plot_freq_fit_pane_model.py

```diff
@@ -0,0 +1,68 @@
+# Mantid Repository : https://github.com/mantidproject/mantid
+#
+# Copyright &copy; 2021 ISIS Rutherford Appleton Laboratory UKRI,
+#   NScD Oak Ridge National Laboratory, European Spallation Source,
+#   Institut Laue - Langevin & CSNS, Institute of High Energy Physics, CAS
+from Muon.GUI.Common.plot_widget.base_pane.base_pane_model import BasePaneModel
+from Muon.GUI.Common.ADSHandler.workspace_naming import *
+
+
+class PlotFreqFitPaneModel(BasePaneModel):
+
+    def __init__(self, context):
+        super().__init__(context,"Frequency Data")
+        self.context.plot_panes_context[self.name].set_defaults([0.,1000.], [-0.3, 0.3])
+
+    @staticmethod
+    def get_fit_workspace_and_indices(fit, with_diff=True):
+        if fit is None:
+            return [], []
+        workspaces = []
+        indices = []
+        for workspace_name in fit.output_workspace_names():
+            first_fit_index = 1  # calc
+            second_fit_index = 2  # Diff
+            workspaces.append(workspace_name)
+            indices.append(first_fit_index)
+            if with_diff:
+                workspaces.append(workspace_name)
+                indices.append(second_fit_index)
+
+        return workspaces, indices
+
+    def _create_workspace_label(self, workspace_name, index):
+        group = str(get_group_or_pair_from_name(workspace_name))
+        run = str(get_run_numbers_as_string_from_workspace_name(workspace_name, self.context.data_context.instrument))
+        instrument = self.context.data_context.instrument
+        fit_label = self._get_fit_label(workspace_name, index)
+        freq_label = self._get_freq_lebel(workspace_name)
+        if not self.context.plot_panes_context[self.name]._is_tiled:
+            return "".join([instrument, run, ';', group, fit_label, freq_label])
+        if self.context.plot_panes_context[self.name].is_tiled_by == "Group/Pair":
+            return "".join([run, fit_label, freq_label])
+        else:
+            return "".join([group, fit_label, freq_label])
+
+    @staticmethod
+    def _get_freq_lebel(workspace_name):
+        label = ''
+        if FFT_STR in workspace_name:
+            label = ''.join([';', get_fft_component_from_workspace_name(workspace_name)])
+        elif MAXENT_STR in workspace_name:
+            label = ''.join([';', MAXENT_STR])
```

Comment:
I would prefer just adding the two string together for speed because we avoid creating a list object, and calling the join method. Or an f string might be easier to read
```suggestion
            label = f';{MAXENT_STR}'
```